src/main/java/bio/terra/app/controller/DatasetsApiController.java

```diff
@@ -194,6 +197,28 @@ public ResponseEntity<SnapshotBuilderGetConceptsResponse> getConcepts(
     return ResponseEntity.ok(snapshotBuilderService.getConceptChildren(id, conceptId));
   }
 
+  @Override
+  public ResponseEntity<DatasetDataModel> queryDatasetDataById(
+      UUID id, String table, QueryDataRequestModel queryDataRequest) {
+    AuthenticatedUserRequest userReq = getAuthenticatedInfo();
+    verifyDatasetAuthorization(userReq, id.toString(), IamAction.READ_DATA);
+    // TODO: Remove after https://broadworkbench.atlassian.net/browse/DR-2588 is fixed
+    SqlSortDirection sortDirection =
+        Objects.requireNonNullElse(queryDataRequest.getDirection(), SqlSortDirection.ASC);
```

Comment:
I wonder if this is still a problem now that this is not a query parameter

Reply:
Great question - After testing this, it appears that it is still needed. If the user doesn't include direction in their request, then it will be null.

Reply:
I've realized that there are some issues with how we're setting our default values. I'm going to have a follow-on PR to address this: https://github.com/DataBiosphere/jade-data-repo/pull/1541